crates/sui-core/src/consensus_validator.rs

```diff
@@ -147,6 +148,27 @@ impl TransactionValidator for SuiTxValidator {
     }
 }
 
+impl TransactionVerifier for SuiTxValidator {
+    fn verify_batch(
+        &self,
+        _protocol_config: &ProtocolConfig,
+        batch: &[&[u8]],
+    ) -> Result<(), ValidationError> {
+        // TODO(mysticeti): ensure we are properly verifying transactions
```

Comment:
Mention what are done improperly here?

Reply:
Forgot to remove the todo, I previously just had this return Ok(). Should be good now as its doing the same thing as the nw tx validator.